pom.xml

```diff
@@ -22,7 +22,7 @@
     <changelist>999999-SNAPSHOT</changelist>
     <gitHubRepo>jenkinsci/${project.artifactId}-plugin</gitHubRepo>
     <!-- remember to change the io.jenkins.tools.bom artifact when changing this -->
-    <jenkins.version>2.277.1</jenkins.version>
+    <jenkins.version>2.346.1</jenkins.version>
```

Comment:
Did you mean 
```suggestion
    <jenkins.version>2.346</jenkins.version>
```
?

Reply:
How could you have tried this PR if the baseline here does not even exist yet? Did you amend the POM afterwards?

Reply:
And why the very new baseline? 2.319.x should suffice for the fix I think? https://github.com/jenkinsci/credentials-plugin/pull/291